spec/system/custom/admin/widgets/cards_spec.rb

```diff
@@ -0,0 +1,30 @@
+require "rails_helper"
+
+describe "Cards", :admin do
+  scenario "Create with title only visible for screen readers" do
+    visit admin_homepage_path
+    click_link "Create card"
+
+    expect(find("#widget_card_display_title")).to be_checked
+    expect(page).to have_field("Display the title?", checked: true)
```

Comment:
The first line seems redundant :thinking:.

```suggestion
    expect(page).to have_field("Display the title?", checked: true)
```